app/models/concerns/reporting_api/event_concern.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ReportingAPI::EventConcern
+  extend ActiveSupport::Concern
+
+  included do
+    belongs_to :source, polymorphic: true
+    belongs_to :patient, optional: true
```

Comment:
This is set to optional, but the column is `not null`.

Reply:
I've removed the `optional: true` now